ceph-releases/luminous/ubuntu/16.04/daemon/demo.sh

```diff
@@ -184,19 +185,75 @@ function bootstrap_mgr {
 }
 
 
+###################
+# BUILD BOOTSTRAP #
+###################
+
+function build_bootstrap {
+  if [[ "$DEMO_DAEMONS" != "all" ]]; then
```

Comment:
In such case, it would be better overriding daemons_list to avoid having two paths of code exec.